dune-project

```diff
@@ -131,6 +131,7 @@ execution of the action.
   (stdune (= :version))
   dyn
   ordering
+  (odoc (>= 2.3.0))
```

Comment:
I don't think It is a good idea to have `dune-glob` and `dune` packages, exclusive regarding `odoc`. I don't know what is behind constraining `odoc` to 2.0.1 in `dune` package.